libmachine/host/version.go

```diff
@@ -0,0 +1,44 @@
+package host
```

Comment:
Why not move the code to get such a `DockerClient` struct to its own module, e.g. `dockerclient`?  There are other parts of the code such as the provisioners which are likely to want to use import this as well.